### ISSUE TITLE
Sandboxed environment?

### ISSUE THREAD

**Issue by jsProj** [engine262/engine262#233]
Is this safe to use as a sandboxed environment? Does anyone know of any methods to escape the contexts/runtime?

**Comment by Jack-Works** [comment]
> Is this safe to use as a sandboxed environment? Does anyone know of any methods to escape the contexts/runtime?

Yes, IMO, but engine262 is not priotizing performance (so the code will be slow). Any capability other than things in the ECMAScript spec must be implemented by you, so the "safety" is up to you.
